lib/simctl.js

```diff
@@ -732,6 +732,57 @@ async function getDeviceTypes () {
   }
 }
 
+
+/**
+ * Get the full list of runtimes, devicetypes, devices and pairs in JSON
+ *
+ * @return {Array<string>} List of the device types, runtimes devices and pairs JSO
+ * @throws {Error} If the corresponding simctl command fails
+ */
+async function getSimctlList () {
+  try {
+    const {stdout} = await simExec('list', 0, ['-j']);
+    /*
+     * JSON will be like
+     *   {
+     *     "devicetypes" : [
+     *       {
+     *         "name" : "iPhone 4s",
+     *         "identifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-4s"
+     *       },
+     *       ...
+     *      ],
+     *     "runtimes" : [
+     *       {
+     *         "version" : '13.0',
+     *         "bundlePath" : '/Applications/Xcode11beta4.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime',
+     *         "isAvailable" : true,
+     *         "name" : 'iOS 13.0',
+     *         "identifier" : 'com.apple.CoreSimulator.SimRuntime.iOS-13-0',
+     *         "buildversion" : '17A5534d'
+     *       },
+     *       ...
+     *      },
+     *     "devices" :
+     *       {
+     *         'com.apple.CoreSimulator.SimRuntime.iOS-13-0': [ [Object], [Object] ] },
+     *         ...
+     *       },
+     *     "pairs" : {} }
+     *
+     *   }
+     */
+    const fullList = JSON.parse(stdout.trim());
+    return fullList;
```

Comment:
This can just be reduced to `return JSON.parse(stdout.trim());`

Reply:
Also, I think the `trim` is unnecessary. It should successfully parse the output without it.